mail_template_conditional_attachment/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_mail_template_conditional_attachment_user,mail.template.conditional.attachment.user,model_mail_template_conditional_attachment,base.group_user,1,1,1,1
```

Comment:
In Odoo, the same permissions are granted to all users on the model, but these are amended by record rules specifying that regular users can only control their own templates: https://github.com/odoo/odoo/blob/18.0/addons/mail/security/mail_security.xml#L265-L285. Can you do the same here (i.e. regular users can only create/write/delete conditional attachment configurations linked to the templates that they own), and editor/manager groups can edit all conditional attachments?